src/main/java/uk/gov/justice/hmpps/prison/api/resource/ScheduleResource.java

```diff
@@ -203,4 +203,11 @@ public List<PrisonerSchedule> getCourtEvents(@PathVariable("agencyId") @Paramete
     public List<PrisonerSchedule> getExternalTransfers(@PathVariable("agencyId") @Parameter(required = true) final String agencyId, @RequestBody @Parameter(description = "The required offender numbers (mandatory)", required = true) final List<String> body, @RequestParam("date") @DateTimeFormat(iso = ISO.DATE) @Parameter(description = "Date of scheduled transfer") final LocalDate date) {
         return schedulesService.getExternalTransfers(agencyId, body, date);
     }
+
+    @Operation
+    @GetMapping("/{prisonerNumber}/scheduledTransfers")
+    @SlowReportQuery
+    public List<PrisonerSchedule> getScheduledTransfersForPrisoner(@PathVariable("prisonerNumber") @Parameter(required = true) final String prisonerNumber) {
```

Comment:
It doesn't look like this endpoint is protected by any roles, am I missing some code somewhere to protect this, I suspect schedule transfers would be *very* sensitive data in high security prisons.

(I'm struggling to see how any of these endpoints are protected on this resource so I must be missing something )

Reply:
That's a fair point and other than some references to `@VerifyAgencyAccess` in the service, doesn't appear to be any protection. This is my first time updating PrisonAPI, so not sure where else to look for confirmation this is correct. I basically copied the similar `getExternalTransfers`